src/compiler/checker.ts

```diff
@@ -1672,6 +1672,14 @@ namespace ts {
                                 return [symbolFromSymbolTable];
                             }
 
+                            // If this is a "default" alias, use what it aliases instead.
+                            if (symbolFromSymbolTable.flags & SymbolFlags.Alias &&
+                                symbolFromSymbolTable.name === "default" &&
+                                symbolFromSymbolTable.declarations.length &&
+                                symbolFromSymbolTable.declarations[0].kind === SyntaxKind.ExportAssignment) {
+                                symbolFromSymbolTable = resolveAlias(symbolFromSymbolTable);

```

Comment:
this is not for visibility. this is for the use of the symbol in qualification, e.g. `Logger.LogLevel` instead of `default.logLevel`